mohammad/CSS/CSSBoxModel/index.html

```diff
@@ -0,0 +1,29 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <div class="row1">
+      <div id="sq1-1"></div>
+      <div id="sq1-2"></div>
+      <div id="sq1-3"></div>
+      <div id="sq1-4"></div>
+      <div id="sq1-5"></div>
+    </div>
+
+    <div class="row2">
+      <div id="sq2-1">Hello</div>
+      <div id="sq2-2">Hello</div>
+      <div id="sq2-3">Hello</div>
+    </div>
+
+    <div class="row3">
```

Comment:
I see a gap between these elements, which is unexpected.

Reply:
I don't have any idea for this
but somehow i fixed it
I know my solution is not very logical
Please tell me the right way🙏